<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="buttonCalculate">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>370</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
background-color: rgb(208, 251, 225);
color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
border-radius: 15px;
border-style: solid;
border-top-color: white;
border-left-color: white;
border-bottom-color: rgb(208, 251, 225);
border-right-color: rgb(208, 251, 225);
border-width: 1px;
}

QPushButton::pressed { 
background-color: rgb(179, 218, 195);
border-style: solid;
border-top-color: rgb(179, 218, 195);
border-left-color: rgb(179, 218, 195);
border-bottom-color: rgb(208, 251, 225);
border-right-color: rgb(208, 251, 225);
border-width: 1px;
}

</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
      </property>
      <property name="text">
       <string>Loan amount</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="inpLoanAmount">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
      </property>
      <property name="text">
       <string>Interest rate</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="inpInterest">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>117</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>117</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
      </property>
      <property name="text">
       <string>Loan term</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="inpTerm">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="buttonMonths">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Months</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonYears">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Years</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
      </property>
      <property name="text">
       <string>Loan type</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="buttonAnnuity">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Annuity</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonDiff">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Differentiated</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 251, 225);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 700 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>Monthly payment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outMonthly">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 700 15pt &quot;Quicksand&quot;;</string>
       </property>
       <property name="text">
        <string>Overpayment </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outOverpayment">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64); 
 font: 700 15pt &quot;Quicksand&quot;;
  padding-right: 1px; </string>
       </property>
       <property name="text">
        <string>Total payment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outTotal">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonClear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
  font: 500 15pt &quot;Quicksand&quot;;
border-radius: 15px;
background-color: rgb(83, 174, 127);
color: rgb(57, 75, 64);
border-style: solid;
border-top-color: rgb(94, 205, 149);
border-left-color: rgb(94, 205, 149);
border-bottom-color: rgb(83, 174, 127);
border-right-color: rgb(83, 174, 127);
border-width: 1px;
}

QPushButton::pressed { 
background-color: rgb(62, 135, 99);
border-style: solid;
border-top-color: rgb(62, 135, 99);
border-left-color:rgb(62, 135, 99);
border-bottom-color: rgb(94, 205, 149);
border-right-color: rgb(94, 205, 149);
border-width: 1px;
}


</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>buttonCalculate</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonClear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
